recipes-debian/openldap/openldap/initscript

```diff
@@ -0,0 +1,35 @@
+#! /bin/sh
+#
+# This is an init script for openembedded
+# Copy it to /etc/init.d/openldap and type
+# > update-rc.d openldap defaults 60
+#
+
+# Source function library.
+. /etc/init.d/functions
+
+slapd=/usr/sbin/slapd
+test -x "$slapd" || exit 0
+
+
+case "$1" in
+  start)
+    echo -n "Starting OpenLDAP: "
+    start-stop-daemon --start --quiet --exec $slapd
```

Comment:
It's related to [this comment](https://github.com/miraclelinux/meta-debian-extended/pull/316/files#r752796170),  execute slapd requires following options.

```
-h "ldap:/// ldapi:///" -F /etc/openldap/ -f /etc/openldap/slapd.conf
```

Reply:
Ah, ``` -f /etc/openldap/slapd.conf ``` doesn't need so ``` -h "ldap:/// ldapi:///" -F /etc/openldap/  ``` would be okay.

Reply:
oops, ``` -F /etc/openldap/slapd.d ``` would be nice.